src/glibc/nptl/allocatestack.c

```diff
@@ -430,18 +431,20 @@ allocate_stack (const struct pthread_attr *attr, struct pthread **pdp,
 	  /* Don't allow setxid until cloned.  */
 	  pd->setxid_futex = -1;
 
-    // BUG: TLS related stuff is not working in lind-wasm currently
-	  /* Allocate the DTV for this thread.  */
-	  // if (_dl_allocate_tls (TLS_TPADJ (pd)) == NULL)
-	  //   {
-	  //     /* Something went wrong.  */
-	  //     assert (errno == ENOMEM);
+    // lind-wasm: glibc's original TLS initialization depends on ELF header
```

Comment:
I'm not sure "clearly not possible" is the right comment, this is more of a TODO possibly since this is what we'll be looking at with dylinking